Source/WebCore/page/EventHandler.cpp

```diff
@@ -4673,11 +4673,16 @@ void EventHandler::updateLastScrollbarUnderMouse(Scrollbar* scrollbar, SetOrClea
 {
     if (m_lastScrollbarUnderMouse != scrollbar) {
         // Send mouse exited to the old scrollbar.
-        if (m_lastScrollbarUnderMouse)
+        if (m_lastScrollbarUnderMouse) {
+            if (auto scrollingCoordinator = m_frame.page()->scrollingCoordinator())
+                scrollingCoordinator->mouseIsInScrollbar(m_lastScrollbarUnderMouse->scrollableArea(), false, m_lastScrollbarUnderMouse->orientation());
```

Comment:
shouldn't this be called `setMouseIsInScrollbar()`?

Reply:
Have you considered passing the scrollbar as parameter instead of both its scrollableArea and its orientation?

Reply:
Fixed.